src/System.AppContext/tests/project.json

```diff
@@ -6,32 +6,22 @@
     "System.ObjectModel": "4.0.12-rc4-24206-03",
     "System.Runtime": "4.1.0-rc4-24206-03",
     "System.Text.RegularExpressions": "4.1.0-rc4-24206-03",
-    "xunit": "2.1.0",
-    "xunit.netcore.extensions": "1.0.0-prerelease-00312-04",
     "test-runtime": {
```

Comment:
The tests still require the test runtime to bring in the the runtime and xunit dependencies in the lock file.